### PR TITLE
VS New PR View / Diffs

### PR DESCRIPTION
**Pull request by CalvinAllen**
Took out the new parts of changing the pathing structure, so this still uses the codestream-diff scheme.

### CODE REVIEW

File: shared/ui/Stream/OpenPullRequests.tsx [hunk 1]
```diff
@@ -311,7 +311,7 @@ export const OpenPullRequests = React.memo((props: Props) => {
 			maximized: settings.maximized,
 			// VS will not use sidebar-diffs currently, uses old method of going directly to
 			// details view when selecting a PR
-			isVS: state.ide?.name?.toUpperCase() === "VS",
+			isVS: false, // state.ide?.name?.toUpperCase() === "VS",
```
Comment (bcanzanella) [inline shared/ui/Stream/OpenPullRequests.tsx:314]: can this and its usage go away?

File: shared/ui/Stream/PullRequest.tsx [hunk 2]
```diff
@@ -141,7 +141,7 @@ export const PullRequest = () => {
 			currentPullRequest: currentPullRequest,
 			currentPullRequestLastUpdated: providerPullRequestLastUpdated,
 			composeCodemarkActive: state.context.composeCodemarkActive,
-			isVsIde: state.ide.name === "VS",
+			isVsIde: false, // state.ide.name === "VS",
```
Comment (bcanzanella) [inline shared/ui/Stream/PullRequest.tsx:144]: can this and its usage go away?

File: shared/ui/Stream/OpenPullRequests.tsx (hunk 1 above)
Comment (CalvinAllen) [reply shared/ui/Stream/OpenPullRequests.tsx:314]: Ah, I lost a commit somewhere along the way of refactoring this thing...

**Review by bcanzanella — CHANGES_REQUESTED** [review]
just those two comments for the `isVS*` stuff. if nothing to do there, this can be merged.

**Pull request merged**